x-pack/test/functional/services/ml/test_resources.ts

```diff
@@ -478,7 +478,7 @@ export function MachineLearningTestResourcesProvider({ getService }: FtrProvider
 
       await retry.tryForTime(30 * 1000, async () => {
         await supertest
-          .post(`/api/fleet/epm/packages/${packageWithVersion}`)
+          .post(`/api/fleet/epm/packages/${packageName}/${version}`)
```

Comment:
If we don't use `packageWithVersion` anymore (see my other comment), we also don't need the definition a few lines above anymore.

Reply:
should be fixed now